cli/src/commands/status.rs

```diff
@@ -72,7 +72,13 @@ pub(crate) fn cmd_status(
                 formatter.labeled("conflict"),
                 "There are unresolved conflicts at these paths:"
             )?;
-            resolve::print_conflicted_paths(&conflicts, formatter, &workspace_command)?
+            resolve::print_conflicted_paths(&conflicts, formatter, &workspace_command)?;
+
+            crate::cli_util::print_conflict_resolution_hint(
```

Comment:
Nit: `cli_util` already is imported so you should call it w/o either as `cli_util::$method` or `$method`, basically dropping the `crate` prefix.

Reply:
done